test/models.jl

```diff
@@ -0,0 +1,21 @@
+@testset "Models logfunctions" begin
+    rng = Random.MersenneTwister(42)
+    D = 5
+    prior = MvNormal(0.5 * ones(D))
+    likelihood = MvNormal(7.0 * ones(D))
+    integrand = x->pdf(likelihood, x)
+    m = BayesModel(prior, integrand)  
```

Comment:
```suggestion
    f = x->pdf(likelihood, x)
    m = BayesModel(p0, f)  
```

`integrand` will clash with the function